RELEASE.md

```diff
@@ -1,7 +1,9 @@
 # Releasing the package
 
 1. Update all the tool versions in [Directory.Build.Props](./Directory.Build.props)
-1. Open a PR to update the `<VersionPrefix>` tag in [Directory.Build.Props](./Directory.Build.props)
+1. Open a PR to update:
+  - the `<VersionPrefix>` tag in [Directory.Build.Props](./Directory.Build.props)
+  - the version `BytecodeAlliance.Componentize.DotNet.Wasm.SDK` package `version` to match the `<VersionPrefix>` i.e. `Version="0.6.0-preview*"` in the [template](./templates/content/wasi-cli/wasi-cli.csproj). This ensures the templates use the latest package.
```

Comment:
looking at this again today, I don't think this simple approach will work since the updated won't exist at the time this is bumped

Reply:
fixed this up so it will use the locally built packages and pass the tests.  This also makes sense for testing the tempaltes with the correct version anyways.